### PR TITLE
Readme copy updates

### PR DESCRIPTION
**Pull request by wmleidy**
Some streamlining, lots of copy editing, and making blackjack = ACE + 10 instead of any combination of 21.

### CODE REVIEW

**Comment by tjbladez** [comment]
:+1:

File: README.markdown [hunk 1]
```diff
@@ -1,63 +1,52 @@
...
-In `accordion.html` you will find ACT course sections breakdown.
-
-Please, write code to simulate the [jQuery accordion](http://docs.jquery.com/UI/Accordion) plugin behavior with the supplied sections.
-By default, the current setup comes with [QUnit](https://qunitjs.com/), feel free to drop and use your own testing framework or write all your code as coffescript.
+## JavaScript/CoffeeScript
 
 ### Simple Backbone.js List
 
-In `quotes.html` you will need to build a simple [Backbone.js](http://backbonejs.org/) view to list given quotes.  Please do not any libraries or plugins to do this other than [jQuery](https://jquery.com/), [Backbone.js](http://backbonejs.org/) and [underscore.js](http://underscorejs.org/) .
+In `quotes.html` you will need to build a simple [Backbone.js](http://backbonejs.org/) view to list the given quotes. Please do not any libraries or plugins to do this other than [jQuery](https://jquery.com/), [Backbone.js](http://backbonejs.org/), and [underscore.js](http://underscorejs.org/).
```
Comment (DrkCloudStrife) [inline README.markdown]: I would change "Please do not any libraries" to "Please do not **use** libraries"

**Comment by nick-desteffen** [comment]
Should we remove the accordion starter code?

File: README.markdown (hunk 1 above)
Comment (wmleidy) [inline README.markdown]: Done

**Comment by wmleidy** [comment]
I can do that. Would make it less confusing. I'll open a separate pull.

**Pull request merged**